rero_ils/modules/documents/views.py

```diff
@@ -323,3 +325,44 @@ def item_status_text(item, format='medium', locale='en'):
             if item.status == ItemStatus.IN_TRANSIT:
                 text += ' ({0})'.format(_(ItemStatus.IN_TRANSIT))
     return text
+
+
+@blueprint.app_template_filter()
+def identifiedby_format(identifiedby):
+    """Format identifiedby for template."""
+    output = []
+    for identifier in identifiedby:
+        status = identifier.get('status')
+        id_type = identifier.get('type')
+        if status not in ('invalid', 'cancelled', 'invalid or cancelled') and \
```

Comment:
Statuses are used only for qualifying bad identifiers. 
So it will be better to write the if statement without the statuses values:
if not status and id_type != 'bf:Local':